.github/workflows/codeforafrica-deploy-review-app.yml

```diff
@@ -0,0 +1,124 @@
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+
+      - name: Clone Database
+        uses: Maggi64/mongo-clone-action@0.3
+        with:
+          source: ${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ secrets.CODEFORAFRICA_MONGO_DB_NAME }}
+          target: ${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ env.APP_NAME }}
+
+      - name: Build Docker image
+        uses: docker/build-push-action@v3
+        with:
+          build-args: |
+            MONGODB_URL=${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ env.APP_NAME }}
+            NEXT_PUBLIC_APP_URL=${{ env.NEXT_PUBLIC_APP_URL }}
+            PAYLOAD_SECRET=${{ secrets.CODEFORAFRICA_PAYLOAD_SECRET_KEY }}
+            GHOST_URL=${{ secrets.GHOST_URL }}
+            GHOST_API_KEY=${{ secrets.GHOST_API_KEY }}
+            NEXT_PUBLIC_APP_LOGO_URL=${{ secrets.NEXT_PUBLIC_APP_CFA_LOGO_URL }}
+            NEXT_PUBLIC_APP_NAME=${{ secrets.NEXT_PUBLIC_APP_CFA_NAME }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
+          context: .
+          file: ./Dockerfile.codeforafrica
+          push: true
+          tags: "${{ env.IMAGE_NAME }}:${{ github.sha }}"
+
+      - name: Push to dokku
+        uses: dokku/github-action@master
+        with:
+          # create a review app
+          command: review-apps:create
+          git_remote_url: ${{ env.DOKKU_REMOTE_URL }}/codeforafrica-ui
+          # specify a name for the review app
+          review_app_name: ${{ env.APP_NAME }}
+          ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+          deploy_docker_image: ${{ env.IMAGE_NAME }}:${{ github.sha }}
+
+      - name: Set Review App Domain Name
+        uses: appleboy/ssh-action@v1.0.0
+        with:
+          host: ui-1.dev.codeforafrica.org
+          key: ${{ secrets.SSH_PRIVATE_KEY }}
+          username: azureuser
+          script: |
+            dokku domains:add ${{ env.APP_NAME }} ${{ env.APP_NAME }}.cms.review.codeforafrica.org --no-restart
+            dokku config:set ${{ env.APP_NAME }} PAYLOAD_PUBLIC_APP_URL=http://${{ env.APP_NAME }}.cms.review.codeforafrica.org --no-restart
+            dokku config:set ${{ env.APP_NAME }} MONGODB_URL=${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ env.APP_NAME }} --no-restart
+
+      - name: Update Preview URL
+        uses: edumserrano/find-create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body-includes: "Latest updated Preview URL"
+          comment-author: "github-actions[bot]"
+          body: |
+            Latest updated Preview URL
+            | Name  | Review |
+            |-------|--------|
+            | ${{ env.APP_NAME }}  | [Visit](http://${{ env.APP_NAME }}.cms.review.codeforafrica.org)  |
+          edit-mode: replace
```

Comment:
Again, can't we just use [`gh`](https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows) directly?

Reply:
@kilemensi we need to achieve

1. Find a comment that matches `Latest updated Preview URL` which gh doesn't.
2. Update the comment with the latest preview URL.

Reply:
Who said `gh` doesn't support that @koechkevin? https://github.com/cli/cli/issues/6790#issuecomment-1376009457